index.html

```diff
@@ -477,7 +477,31 @@ <h3>Required Owned Elements</h3>
 		</section>
 		<section id="scope">
 			<h3>Required Context Role</h3>
-			<p>The required context role defines the owning container where this <a>role</a> is allowed. If a role has a required context, authors MUST ensure that an element with the role is contained inside (or <a>owned</a> by) an element with the required context role. For example, an element with role <code>listitem</code> is only meaningful when contained inside (or owned by) an element with role <code>list</code>.</p>
+			<p>The required context role defines the owning container where this <a>role</a> is allowed. If a role has a required context, authors MUST ensure that an element with the role is contained inside (or <a>owned</a> by) an element with the required context role, with no other intermixed containing elements with non-presentational roles.</p>
```

Comment:
What exactly are "elements with non-presentational roles"? This term is only ever used as "explicit non-presentational role", which is clear. But for the implicit case this could be misunderstood. It can either mean it has no non-presentional role in ARIA, or it has no non-presentational role in the accessibility API.

The `canvas` element is one that comes to mind. There is no ARIA role that maps to `canvas`, so in that sense it would not count for the required context. But if you consider what role it is [mapped to in the accessibility API](https://w3c.github.io/html-aam/#el-canvas), canvas does have a non-presentational role.

Reply:
Yes.  That does need more clarity if it's still unclear that one shouldn't be inserting improper markup and roles between the element/role and its required context role.

Essentially needs to be revised to call out `role=presentation` and `none` are fine, anything else is not.

Reply:
I've made further updates to this text to address your comments, @WilcoFiers 

preview still doesn't seem to be working, but not getting build failures anymore at least.